website/project/model.py

```diff
@@ -635,6 +635,11 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
 
     creator = fields.ForeignField('user', backref='created')
     contributors = fields.ForeignField('user', list=True, backref='contributed')
+    # Time of last sent notification email to newly added contributors
+    # {<User._id>:
+    #   {'last_sent': time.time()}
+    # }
+    contributor_record = fields.DictionaryField(default=dict)
```

Comment:
What was the rationale for adding this to the `Node` model?

It seems more consistent to add this data to the `User` model (see the `security_messages` and `unclaimed_records` fields).

Also, the meaning of "contributor_records" is a bit ambiguous; could be more specific, like "contributor_added_email_records"?